numba_cuda/numba/cuda/__init__.py

```diff
@@ -44,6 +46,16 @@
             "bindings."
         )
 
+if config.CUDA_ENABLE_PYNVJITLINK:
+    if USE_NV_BINDING:
+        warnings.warn(
+            "Explicitly enabling PyNvJitLink no longer necessary. "
```

Comment:
We've never written this name in CamelCase, just all lowercase.
```suggestion
            "Explicitly enabling pynvjitlink is no longer necessary. "
```